src/game.cpp

```diff
@@ -157,6 +157,9 @@ void Game::setGameState(GameState_t newState)
 	}
 }
 
+#if !defined(_MSC_VER)
```

Comment:
My understanding is that `__attribute__` is GCC specific and I assumed without the def an error would occur when compiling in visual studio. If my understanding or assumption is incorrect, it can be removed.